integration_tests/src/main/python/map_test.py

```diff
@@ -32,6 +38,39 @@ def test_simple_get_map_value(data_gen):
                 'a["NOT_FOUND"]',
                 'a["key_5"]'))
 
+@pytest.mark.parametrize('key_gen', [StringGen(nullable=False), IntegerGen(nullable=False), basic_struct_gen], ids=idfn)
+@pytest.mark.parametrize('value_gen', [StringGen(nullable=False), IntegerGen(nullable=False), basic_struct_gen], ids=idfn)
```

Comment:
Why no nulls on the values?

Reply:
And also the keys?

Reply:
Spark doesn't allow nulls for the keys. The tests have been updated to allow nulls in the values.